### PR TITLE
Update / clarify / shorten docs, add missing image from AWS install

### PR DESCRIPTION
**Pull request by laxdog**
- [x] Add / update documentation
 - [N/A] Add tests

 

To be honest, I'm still not sure about the CPU part. The original didn't make any sense, so I tried to reword it to something that did. However, while running my own set up of this, I've not seen much of a CPU overhead.

### CODE REVIEW

File: docs/install/amazon.rst [hunk 1]
```diff
@@ -84,21 +83,18 @@ Let's create the server on which we can run JupyterHub.
...
-   developing this tutorial using  **t2.micro** tier. That tier is free for 
-   Amazon users the first year they sign up. Two users were able to concurrently 
-   access this development hub.)
+   (For reference, a minimal hub that worked for developing this tutorial used a
+   **t2.micro** tier, which is free for Amazon users the first year they sign
+   up. Two users were able to concurrently utilize this development hub without issue.)
    
-   If you want to **GPUs**, you'll have to scroll down about half way to find 
-   the `GPU graphics` and `GPU compute` products.
+   `GPU graphics` and `GPU compute` products are also available around half way down the page
```
Comment (choldgraf) [inline docs/install/amazon.rst]:
```suggestion
   ``GPU graphics`` and ``GPU compute`` products are also available around half way down the page
```

File: docs/howto/admin/resource-estimation.rst [hunk 2]
```diff
@@ -58,11 +59,13 @@ stop, unlike with RAM.
 
 .. math::
 
-    Server CPU Recommended = (Maximum concurrent users \times Maximum CPU usage per user) + 0.2
+    Recommended\, CPU = (Max\, concurrent\, users \times Max\, CPU\, usage\, per\, user) + 20\%
```
Comment (choldgraf) [inline docs/howto/admin/resource-estimation.rst]: I'm a little unsure how the commas will render here (will they render)?

File: docs/contributing/docs.rst [hunk 3]
```diff
@@ -60,6 +60,12 @@ Then from the ``docs`` directory, build the HTML:
 
      $ make html
 
+If you encounter this error, it's likely that running it in virtual environment is your problem.
```
Comment (leportella) [inline docs/contributing/docs.rst]: What do you think of changing to  `it's likely that you are running inside a virtual environment` ?

File: docs/howto/admin/resource-estimation.rst [hunk 4]
```diff
@@ -58,11 +59,13 @@ stop, unlike with RAM.
...
 
-The ``0.2`` is overhead for TLJH and related services. **Server CPU Recommended**
-is the amount of CPU the server you acquire should have. We recommend using
+The ``20%`` is overhead for TLJH and related services. This is around 20% of a 
+single modern CPU. This of course is just an estimate. We recommend using
```
Comment (leportella) [inline docs/howto/admin/resource-estimation.rst]: I think that a couple of commas are missing: 

`This, of course, is just an estimate`

**Comment by laxdog** [comment]
Just noticed these comments. I'll have a proper look this weekend.

File: docs/howto/admin/resource-estimation.rst (hunk 2 above)
Comment (laxdog) [reply docs/howto/admin/resource-estimation.rst]: They render to leave spaces. The previous code didn't render correctly
Without commas:
Screenshot 2019-04-14 at 21 19 10

With commas:
Screenshot 2019-04-14 at 21 17 44

File: docs/contributing/docs.rst (hunk 3 above)
Comment (laxdog) [reply docs/contributing/docs.rst]: Yea, that sounds much better.

**Comment by choldgraf** [comment]
wohoo! 🎉
